tools/boostsecurity

```diff
@@ -0,0 +1,69 @@
+{
+      "name": "Boostsecurity.io",
+      "publisher": "Boostsecurity.io",
+      "description": "The zero friction platform for AppSec Testing and Application Security Posture Management (ASPM) securing both Software Supply Chains and AI Development.",
+      "website_url": "https://boostsecurity.io/",
+      "capabilities": [
+        "SBOM"
+      ],
+      "availability": [
+        "COMMERCIAL_LICENSE",
+        "SUBSCRIPTION"
+      ],
+      "functions": [
```

Comment:
could you elaborate on these functions? the tool's current description nor the website dont really have information regarding  these claims.